src/Repository/TagsRepository.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\Tags;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\ORM\NonUniqueResultException;
+use Doctrine\Persistence\ManagerRegistry;
+
+/**
+ * @extends ServiceEntityRepository<Tags>
+ *
+ * @method Tags|null find($id, $lockMode = null, $lockVersion = null)
+ * @method Tags|null findOneBy(array $criteria, array $orderBy = null)
+ * @method Tags[]    findAll()
+ * @method Tags[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class TagsRepository extends ServiceEntityRepository
+{
+    public function __construct(ManagerRegistry $registry)
+    {
+        parent::__construct($registry, Tags::class);
+    }
+
+    public function save(Tags $entity, bool $flush = false): void
+    {
+        $this->getEntityManager()->persist($entity);
+
+        if ($flush) {
+            $this->getEntityManager()->flush();
+        }
+    }
+
+    public function remove(Tags $entity, bool $flush = false): void
+    {
+        $this->getEntityManager()->remove($entity);
+
+        if ($flush) {
+            $this->getEntityManager()->flush();
+        }
+    }
+
+    /**
+     * @param $label
+     * @throws NonUniqueResultException
+     */
+    public function findOneByLabelCI(string $label): ?Tags
```

Comment:
What "CI" means?

Reply:
Why do you have to name function like this? Is "case insensitive" is not always required?

Reply:
> Why do you have to name function like this?
I prefer to specify to avoid surprises.

> Is "case insensitive" is not always required?
If only, Win versus Linux